src/main/java/org/eln2/mc/common/CommonEvents.kt

```diff
@@ -38,4 +52,28 @@ object CommonEvents {
             }
         }
     }
+
+    @SubscribeEvent
+    fun onEntityJoinedWorld(event: EntityJoinWorldEvent) {
+        //Warn new users that we collect analytics
+        if (event.world.isClientSide && event.entity is Player) {
```

Comment:
So we're going to likely have both server and client side analyitcs so we're likely going to want to have _two_ configs eventually.